### ISSUE TITLE
feat: support pass-through config from Meltano

### ISSUE THREAD

**Issue by kgpayne** [MeltanoLabs/evidence-ext#1]
Similar to how the [airflow-ext](https://github.com/meltano/airflow-ext) uses the EDK [pre_invoke hook](https://github.com/meltano/airflow-ext/blob/main/airflow_ext/wrapper.py#L52) to create a config file on disk, we could take a similar approach to create the `evidence/template/evidence.settings.json` file. This would allow the management and injection of config from Meltano (including per Meltano Environment etc.) 🚀

**Comment by kgpayne** [comment]
@archiewood are there any resources you can point to for the expected schema of this file? The duckDB sample project produces:

```json
{
  "database": "duckdb",
  "credentials": {
    "filename": "needful_things.duckdb",
    "gitignoreDuckdb": null
  }
}
```

I am happy to poke around with the other supported DB's to figure it out but you might know a better way 😅

**Comment by kgpayne** [comment]
Thanks @archiewood that helps a lot 🙏 One thing I did notice is that, although the env vars are picked up in `dev`, certain UI elements in Settings don't function the way I'd expect. I.e. the Settings page behaves as though no connection is specified, even though the actual queries work fine in all the other pages. Its a very minor rough edge though - very chuffed the env vars approach appears to mostly "just work" 🙌

**Comment by archiewood** [comment]
Yeah I also noticed that. I think that's a pretty minor rough edge as you say. 
Something we can look into fixing if this is a common pattern!

**Comment by kgpayne** [comment]
@archiewood smashing, thanks again for your help 🙌

**Comment by kgpayne** [comment]
Closed by #2 and #4
